test/hamiltonian_consistency.jl

```diff
@@ -7,6 +7,27 @@ include("testcases.jl")
 using Random
 Random.seed!(0)
 
+function test_matrix_repr_opererator(block, ψ, Hψ; atol=1e-8)
```

Comment:
Our convention here is `hamk` and `ψk` as well as `Hψk`

Reply:
I have put `δψkδHψk` also, but it's a bit a mouthful.